resources/assets/components/Users/UserInformation.js

```diff
@@ -14,7 +14,20 @@ const UserInformation = props => (
         }
         <p>
           {props.user.email ? <span>{props.user.email}<br /></span> : null}
-          {props.user.mobile ? <span>{props.user.mobile}<br /></span> : null }
+
+          {
+            (() => {
+              if (props.user.mobile) {
+                if (props.linkSignup && isEmpty(props.user.first_name)) {
+                  return (<span><a href={`/signups/${props.linkSignup}`}>{props.user.mobile}</a><br /></span>);
+                } else {
```

Comment:
I think you might not need this else and can just return `(<span>{props.user.mobile}<br /></span>)` here since the `if` returns something as well

Reply:
smart!